docs/general/performance.md

```diff
@@ -34,6 +34,31 @@ new Chart(ctx, {
 });
 ```
 
+## Specify `min` and `max` to scales
```

Comment:
nit: just noticed this. "on scales" or "for scales" would be more common

Reply:
Changed